presto-main/src/main/java/io/prestosql/operator/aggregation/ApproximateDoublePercentileAggregations.java

```diff
@@ -54,6 +54,19 @@ public static void weightedInput(@AggregationState DigestAndPercentileState stat
         ApproximateLongPercentileAggregations.weightedInput(state, doubleToSortableLong(value), weight, percentile, accuracy);
     }
 
+    // #758 -- weight can be double as well as long
```

Comment:
This comment is unnecessary. Maybe add a short explanation to the commit message about the overall high-level approach.